src/nft-card-back.ts

```diff
@@ -228,7 +243,8 @@ export class NftCardBackTemplate extends LitElement {
   }
 
   public getStatsTemplate(stats) {
-    return html`
+  if (stats.length <= 0) { return } // Don't render if empty array
```

Comment:
this was in my previous PR review, but let's explicitly return undefined here, and also do if statements on at least 3 lines. makes them much more readable